lib/rake/eyes.rake

```diff
@@ -34,17 +34,27 @@ end
 
 def force_merge_eyes_baselines(branch, base)
   ensure_merge_util
-  RakeUtils.system_stream_output "#{BASE_MERGE_UTIL_CALL} -n Code.org -o -s #{branch} -t #{base}"
+  RakeUtils.system_stream_output "#{BASE_MERGE_UTIL_CALL} -n Code.org -o true -s #{branch} -t #{base}"
```

Comment:
Is this an idiosyncrasy of the merge util API?  I guess they don't support shorthand flags `-abc`.